### ISSUE TITLE
[Feature Request] Integration idea with Tachiyomi

### ISSUE THREAD

**Issue by Syer10** [Suwayomi/Suwayomi-Server#159]
# Integration with Tachiyomi through a source and a tracker

## What feature should be added to Tachidesk?
Ideas:
- [x] Have a source for Tachiyomi that looks through your Tachidesk library for manga like Komga
- [x] Have a tracker in Tachiyomi that acts like the Komga tracker does, and brings read status to and from Tachidesk
- [ ] Track categories can be the categories it is in inside Tachidesk, which can be loaded at runtime
- [x] When refreshing the manga, load the new chapters and info from the source in Tachidesk
- [ ] The source can load filters are runtime, so you can categorize your manga and have the categories be a filter option

##  Why/Project's Benefit/Existing Problem
Will allow a lightweight way of syncing with Tachiyomi

# Implementation of SyncYomi support in the Server

## What feature should be added to Tachidesk?

Ideas:
- [x] Implement client support for SyncYomi server
- [ ] Implement a SyncYomi server API and so we can pass as a SyncYomi server

##  Why/Project's Benefit/Existing Problem

SyncYomi was created to work as a self-hosted sync server for Tachiyomi/Mihon, it achieves some features that we want in Suwayomi. Refer to #1102 for tracking progress.

**Comment by vtruong68** [comment]
If you're having trouble connecting to your local Tachidesk server with the Tachiyomi extension, try disabling DNS over HTTPS if it's enabled (Tachiyomi Settings > Advanced > DNS over HTTPS).

The error says something like NXDOMAIN since it can't find your LAN ip.

**Comment by itsmefdil** [comment]
I think need feature for download some chapter or all chapter like tachiyomi .

**Comment by kanjieater** [comment]
Yes - this would also be a place that could benefit from the page syncing request here https://github.com/tachiyomiorg/tachiyomi/issues/5978

**Comment by AriaMoradi** [comment]
> Yes - this would also be a place that could benefit from the page syncing request here [tachiyomiorg/tachiyomi#5978](https://github.com/tachiyomiorg/tachiyomi/issues/5978)

That would be a very nice thing to have, but for now the Tachidesk tracker pull request https://github.com/tachiyomiorg/tachiyomi/pull/8489 has been waiting for a merge for almost a month. It'll be probably sitting there for more time for reasons. We can only hope someday tachiyomi people will stop hating on Tachidesk.

**Comment by NotTsunami** [comment]
> > Yes - this would also be a place that could benefit from the page syncing request here [tachiyomiorg/tachiyomi#5978](https://github.com/tachiyomiorg/tachiyomi/issues/5978)
> 
> That would be a very nice thing to have, but for now the Tachidesk tracker pull request [tachiyomiorg/tachiyomi#8489](https://github.com/tachiyomiorg/tachiyomi/pull/8489) has been waiting for a merge for almost a month. It'll be probably sitting there for more time for reasons. We can only hope someday tachiyomi people will stop hating on Tachidesk.

Outsider here but that last comment is only instigating, rather intentional or not. You were simply asked to rename the project due to the similar naming convention. You responded with "*My team* and I determined this was not necessary." This does not change the fact you were simply asked to change the name, nor does it mean that the Tachiyomi organization hates you. There was no hatred exhibited in the PR either.

**Comment by AriaMoradi** [comment]
> Outsider here but that last comment is only instigating, rather intentional or not. You were simply asked to rename the project due to the similar naming convention. You responded with "My team and I determined this was not necessary." This does not change the fact you were simply asked to change the name, nor does it mean that the Tachiyomi organization hates you. There was no hatred exhibited in the PR either.

We talked about it on their discord some date in November, We came into a compromise which was naming the tracker "Suwayomi". All issues should have been resolved. But there's one little issue that Suwayomi the Tracker references the Tachidesk Extension (because it needs to link to it in the code) and there's no way to fix that.  If there are no further issues then the PR should be merged into Tachiyomi, but it might be as late as 0.15.x or 0.16.x as they didn't merge the [Kavita Tracker](https://github.com/tachiyomiorg/tachiyomi/pull/7488) until 0.14.x while it was created in the 0.12.x lifetime

The issue of hating on Tachidesk is because we named the app Tachidesk (there was a poll to rename it but the majority voted to keep the name) and also some people from the Tachiyomi community believe that we are leeching users and popularity from Tachiyomi. As a side note the predecessor to Tachides was named [TachiWeb](https://github.com/TachiWeb/TachiWeb-Server) and apparently no one had any issues with that name...

 Anyways, Take what you see as fair.

**Comment by NotTsunami** [comment]
> We talked about it on their discord some date in November, We came into a compromise which was naming the tracker "Suwayomi".

That's something I missed when reading through this. My apologies there.

> The issue of hating on Tachidesk is because we named the app Tachidesk (there was a poll to rename it but the majority voted to keep the name) and also some people from the Tachiyomi community believe that we are leeching users and popularity from Tachiyomi.

This seems like a crappy situation to be in for either party involved, because it is impossible to please both sides. I looked at this from one perspective only and I apologize for any hostility the original comment harbored. 

Hope you had a wonderful Christmas if you celebrate and good luck moving forward with the pull request.

**Comment by ghost** [comment]
there is this app called [syncyomi ](https://github.com/SyncYomi/SyncYomi)which use local server to sync multiple devices (using modified tachiyomi apk) including read status, new manga addition, etc. The database stored in server locally. maybe devs can take a look at this..

**Comment by Tsukuyomit65** [comment]
Yo uso tachiyomi en el celular, hay alguna forma de instalarlo para celulares?
